libtenzir/builtins/operators/load_tcp.cpp

```diff
@@ -256,12 +284,20 @@ class load_tcp_sink_operator final
   }
 
   friend auto inspect(auto& f, load_tcp_sink_operator& x) -> bool {
-    return f.apply(x.connection_manager_);
+    return f.object(x).fields(
+      f.field("connection-manager", x.connection_manager_),
+      f.field("peer-field", x.peer_field_), f.field("peer-ip", x.peer_ip_),
+      f.field("peer-port", x.peer_port_),
+      f.field("peer-hostname", x.peer_hostname_));
```

Comment:
Let's be consistent with the load_tcp_args field names and use `_` instead of `-` everywhere.